### ISSUE TITLE
httpexample javascript files not found on readthedocs

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden#1896]
## Problem Definition

The HTTP snippets in the docs are  rendered incorrectly:
![screen shot 2018-07-20 at 11 13 22](https://user-images.githubusercontent.com/90851/42994304-05fa3564-8c0e-11e8-9ab1-d9a100788bd5.png)

**Comment by palango** [comment]
My browser console shows the following: We don't seem to provide the correct javascript file.
```
Loading failed for the <script> with source “https://raiden-network.readthedocs.io/en/latest/_static/sphinxcontrib-httpexample.js”.
```

**Comment by palango** [comment]
Fixed with #1909
